spec/requests/events/show_spec.rb

```diff
@@ -16,10 +16,14 @@
       end
 
       it 'returns current users event info' do
-        get "/events/#{event.id}"
-        expect(response.body).to include(event.name)
-        expect(response.body).to include(event.scheduled_for.to_s)
-        expect(response.body).to include(event.category.name)
+        VCR.use_cassette('fixtures') do
+          get "/events/#{event.id}"
+          expect(response.status).to eq(200)
+          expect(response.body).to include(event.name)
+          expect(response.body).to include(event.scheduled_for.to_s)
+          expect(response.body).to include(event.category.name)
+          expect(response.body).to include("Weather for #{event.scheduled_for.strftime('%Y-%m-%d')}")
+        end
```

Comment:
You need to use self-describing names for cassettes. Name it `forecast/response` or something like that.

```suggestion
        VCR.use_cassette('forecast/response', match_requests_on: [:headers, :body, :uri]) do
          get "/events/#{event.id}"
        end
          
        expect(response.status).to eq(200)
        expect(response.body).to include(event.name)
        expect(response.body).to include(event.scheduled_for.to_s)
        expect(response.body).to include(event.category.name)
        expect(response.body).to include("Weather for #{event.scheduled_for.strftime('%Y-%m-%d')}")
```